matscipy/precon.py

```diff
@@ -0,0 +1,51 @@
+"""Hessian preconditioner."""
+
+import numpy as np
+
+from ase.optimize.precon import Precon
+from ase.calculators.calculator import equal
+
+
+class HessianPrecon(Precon):
+    """Preconditioner for dense Hessian."""
+
+    def __init__(self,
+                 calc,
```

Comment:
Is there a reason why the preconditioner has to have a `calc` member? Since an `atoms` object is passed on to `make_precon`, shouldn't it use `atoms.calc` instead?

Reply:
That's a fair comment, happy to change this